includes/discovery/os/sharpprinter.inc.php

```diff
@@ -0,0 +1,7 @@
+<?php
+if (!$os) {
+    // Sharp Multifunction Printer/Scanner
+    if (strstr($sysDescr, 'SHARP MX-')) {
```

Comment:
We have custom functions to deal with this, you should update this to:

`if (str_contains($sysDescr, 'SHARP MX-')) {`